docs/src/dev-guide/tooling-containers.md

```diff
@@ -1,34 +1,110 @@
 # Containers
 
-We publish (to [GitHub packages][gh-packages]) several Docker container images useful for building
-and running CLP:
+We maintain several Docker container images that are useful for building and running CLP. All images
+can be built and used locally, but some are available to download from
+[GitHub Packages][gh-packages].
 
-* An [image][core-deps-centos-stream-9] containing the dependencies necessary to build CLP core in a
-  Centos Stream 9 x86 environment.
+To build an image locally, run the `build.sh` script in the image's directory.
+
+## clp-core-dependencies-&lt;arch&gt;-manylinux-2.28
+
+Images containing the dependencies necessary to build CLP core in a manylinux 2.28 environment
+(aarch64 or x86).
+
+Binaries built on manylinux 2.28 (based on AlmaLinux 8) are expected to be compatible with other
+distros using glibc 2.28+, including:
+
+* CentOS/RHEL 8+
+* Debian 10+
+* Fedora 29+
+* Ubuntu 18.10+
+
+### clp-core-dependencies-aarch64-manylinux-2.28
```

Comment:
@coderabbitai, can you open an issue for renaming our container images to be more consistent? I propose:

* the format `<image-name>-<os-name>-<os-version>-<architecture>`.
* using "x86_64" to replace wherever we use "x86".
* when we eventually build multiplatform images, we can drop the `-<architecture>` part.